src/Aspose.Imaging.Cloud.Sdk.Test/Base/ApiTester.cs

```diff
@@ -228,6 +228,7 @@ protected void CreateApiInstances()
 
             if (serverFileInfo.Exists && serverFileInfo.Length > 0)
             {
+                WriteLineEverywhere($"!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!");
```

Comment:
@dmitrypasyutin It looks like an overlooked debug message, isn't it?

Reply:
@s-zubov it really is, will remove it now